src/y0/algorithm/counterfactual_transport/graveyard.py

```diff
@@ -0,0 +1,213 @@
+"""Unused functions implemented for CFT."""
```

Comment:
@rjc955 please check that nothing in this file is needed, then delete it. This code isn't used anywhere.